components/reference-data/src/main/java/org/datacleaner/beans/transform/RemoveDictionaryMatchesTransformer.java

```diff
@@ -59,9 +65,14 @@
     @Configured(value = PROPERTY_COLUMN)
     InputColumn<String> _column;
 
+    @Inject
+    @Configured(required = false)
+    @Description("Get removed strings as List")
+    boolean _removedMatchesAsList = false;
```

Comment:
Could we use an enum for this instead? Would be slightly more intuitive (and future-proof) to select between multiple strategies. And I would like the default to be list.